### PR TITLE
Adding Documentation for Solving Linear Program with OMMX Adapter

### PR DESCRIPTION
**Pull request by Shitaro**
This pull request introduces a Japanese tutorial on solving a 0-1 knapsack problem using the OMMX PySCIPOpt Adapter.

To emphasize the use of the OMMX Adapter, I renamed the placeholder page `solve_with_scip` to `solve_with_ommx_adapter`. Additionally, to accommodate an image required for the tutorial, I created an `assets` directory under the `doc/tutorial` directory.

### CODE REVIEW

**Comment by Shitaro** [comment]
I revised the text in https://github.com/Jij-Inc/ommx/pull/171/commits/7700b58133c4eebad8ffa28e14c756fa9f857830. In the process, I discovered that there are multiple optimal solutions in the knapsack problem data used in this tutorial. This could complicate the results of the Python tests, so I'll modify the data to ensure that there is only one optimal solution.

**Comment by Shitaro** [comment]
@NY57 
In the commit https://github.com/Jij-Inc/ommx/pull/171/commits/1bb78cf281c14104c4142098de45628c8ab763d0, I modified the data so that the instance of the 0-1 knapsack problem has only one optimal solution.

File: doc/tutorial/assets/solve_with_ommx_adapter_01.png
Comment (NY57) [inline doc/tutorial/assets/solve_with_ommx_adapter_01.png:1]: 画像内の`w`が更新できていないようです。更新よろしくおねがいします！

**Review by NY57 — CHANGES_REQUESTED** [review]
1点だけ不備がありましたので修正をお願いします！

File: doc/tutorial/assets/solve_with_ommx_adapter_01.png
Comment (Shitaro) [reply doc/tutorial/assets/solve_with_ommx_adapter_01.png:1]: fix in https://github.com/Jij-Inc/ommx/pull/171/commits/7169e81517cdce81320474cfa28741c73e016fe9

**Review by NY57 — APPROVED** [review]
LGTM!

**Comment by Shitaro** [comment]
@NY57 
Oops, I changed the data but it wasn't reflected in the figure.
With the commit https://github.com/Jij-Inc/ommx/pull/171/commits/7169e81517cdce81320474cfa28741c73e016fe9, I updated the figure.

image

**Pull request merged**